cmd/kubeadm/app/util/kustomize/kustomize.go

```diff
@@ -207,7 +208,7 @@ func (km *Manager) Kustomize(data []byte) ([]byte, error) {
 		return nil, err
 	}
 	name := "resource.yaml"
-	_ = memFS.WriteFile(filepath.Join(fakeDir, name), b)
+	memFS.WriteFile(filepath.Join(fakeDir, name), b)
```

Comment:
I searched the code base, and it seemed that the return values can be ignored.
e.g. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/kustomize/builder_test.go#L136